src/lib/rest/rest.cpp

```diff
@@ -569,6 +569,8 @@ static void requestCompleted
   std::string      spath    = (ciP->servicePathV.size() > 0)? ciP->servicePathV[0] : "";
   struct timespec  reqEndTime;
 
+  // compoundSaveReport();
```

Comment:
If this comment is preserved at the end (seems to be a good idea), a FIXME mark citing #3162 should be included in the line above it.